StabQuest/Services/SoundService.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.Xna.Framework.Media;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Audio;
+using System.Collections.Generic;
+
+namespace StabQuest.Services;
+
+public sealed class SoundService
+{
+    private static readonly SoundService _instance = new();
+
+    private Song _backgroundSong1;
+    private Song _backgroundSong2;
+    private List<SoundEffect> _soundEffects;
+    private bool _isBackgroundSong1Playing;
+    private int stepCounter;
+
+    static SoundService() { }
+
+    private SoundService() { }
+
+    public static SoundService Instance
+    {
+        get
+        {
+            return _instance;
+        }
+    }
+
+    public ContentManager Content { get; set; }
+
+    public enum SoundEffects
+    {
+        Sword,
+        Step,
+    }
```

Comment:
Maybe the different sound effects should be their own classes with their own "Play" function or something? I'm thinking like maybe you want to have a few different "sword swing" sounds? Postpone for future perhaps?